x-pack/plugins/security_solution/public/resolver/store/middleware/resolver_tree_fetcher.ts

```diff
@@ -91,6 +92,25 @@ export function ResolverTreeFetcher(
             parameters: databaseParameters,
           },
         });
+
+        /*
+         * Necessary to handle refresh states where another node besides the origin was selected
+         * If the user has selected another node, but is back to viewing the node list
+         * Refresh will reset to the originID.
+         * This is okay for now, but can be updated if we decide to track selectedNode in panelParameters.
+         */
+        if (currentPanelParameters && currentPanelParameters.panelView !== 'nodes') {
```

Comment:
Took me a second to figure out why we were skipping this when `panelView !== 'nodes'` It's because if that's true it means we're on the process list page in the panel, therefore they must not have selected a node.

Maybe add a comment about that 🤷